quarry/types/nbt.py

```diff
@@ -262,6 +265,15 @@ def to_mojangson(self, sort=None, highlight=False):
             inner_mojangson.append(f'{content!s}{type_postfix}')
         return f'{prefix}{separator.join(inner_mojangson)}{postfix}'
 
+    def to_json(self):
+        inner_json = []
+        for content in self.value:
+            # Converted packed unsigned values to signed values
+            if content >= (1<<(self.width - 1)):
+                content -= (1<<(self.width))
```

Comment:
I suspect we don't want to change signed data to unsigned in most cases.

Reply:
Nevermind, re-read it after Combustible pointed out the other method does the same thing.